File: mutation.ts
const mutation = (strings: string[]) => {
  if (!Array.isArray(strings) || strings.length !== 2)
    throw new Error("input must be a valid array of string")

  const [source, target] = strings;
  return target.split("").every(s => source.includes(s))
}

console.log(mutation(["Alien", "lien"]))
console.log(mutation(["Hello", "hey"]))

File: mutation.py
def mutation(strings: list[str]) -> bool:
    source, target = [s.lower() for s in strings]
    return all(char in source for char in target)


print(mutation(["Alien", "lien"]))  # true
print(mutation(["Alien", "Lien"]))  # true
print(mutation(["Hello", "hey"]))  # false
